mixer3: {
  none: none,
  RGB: /usr/lib/frei0r-1/RGB.so,
}
